code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -134,4 +143,12 @@
 	..()
 	// Change my name back, don't want to be named Tom (666)
 	name = initial(name)
-	real_name = name
\ No newline at end of file
+	real_name = name
+
+/mob/living/simple_animal/mouse/fat

```

Comment:
Unless I'm misreading, fatmouse has a 2% chance to be remade in the image of fatmouse, despite already being a fat mouse.